client/landing/gutenboarding/components/domain-picker-popover/index.tsx

```diff
@@ -15,16 +15,21 @@ import DomainPicker, { Props as DomainPickerProps } from '@automattic/domain-pic
  */
 import './style.scss';
 
-interface Props extends Omit< DomainPickerProps, 'tracksName' > {
+interface Props extends DomainPickerProps {
 	isOpen: boolean;
+	onOpen: () => void;
 }
 
-const DomainPickerPopover: React.FunctionComponent< Props > = ( { isOpen, ...props } ) => {
+const DomainPickerPopover: React.FunctionComponent< Props > = ( { isOpen, onOpen, ...props } ) => {
 	const onClose = props.onClose;
 
 	// Popover expands at medium viewport width
 	const isMobile = useViewportMatch( 'medium', '<' );
 
+	React.useEffect( () => {
+		isOpen && onOpen && onOpen();
+	}, [ isOpen, onOpen ] );
```

Comment:
🙈 I hope we don't need to do this and fire open event directly from this component. https://github.com/Automattic/wp-calypso/pull/42659/files#r430891845

Reply:
Yep, makes sense. It's gonna go in there. https://github.com/Automattic/wp-calypso/pull/42659#discussion_r430927435